### PR TITLE
Update version number to 1.22.0.

### PR DESCRIPTION
**Pull request by jhoole**
Referenced updates made for 1.21: https://github.com/chapel-lang/chapel/compare/512d4a03d0d6...e6731d8a2509

### CODE REVIEW

File: doc/rst/language/archivedSpecs.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ Documentation Archives
 Online Documentation Archives
 -----------------------------
 
+* `Chapel 1.21 <https://chapel-lang.org/docs/1.20/index.html>`_
```
Comment (ronawho) [inline doc/rst/language/archivedSpecs.rst:9]: Link points to 1.20

Comment (jhoole) [reply doc/rst/language/archivedSpecs.rst:9]: Thanks!

**Comment by bradcray** [comment]
A test run suggests that test/compflags/bradc/printstuff/version.goodstart was missed.

Comparing against PR #14124, I didn't notice any other files that had been missed.

**Comment by jhoole** [comment]
Ah, it was already set to 1.21.0 which is why it didn't get updated in the last PR. Updated now.

**Comment by bradcray** [comment]
Those tests came back clean after that change.  I think this is ready to merge.

**Pull request merged**